pkg/authentication/scram/scram.go

```diff
@@ -43,6 +44,9 @@ type Configurable interface {
 
 	// NamespacedName returns the NamespacedName for the resource that is being configured.
 	NamespacedName() types.NamespacedName
+
+	// GetOwnerReferences returns the OwnerReference pointing to the current resource.
```

Comment:
```suggestion
	// GetOwnerReferences returns the OwnerReferences pointing to the current resource.
```

Reply:
Although we actually return exactly one, so maybe not even the right suggestion :D up to you!

Reply:
changed